chapters/sampling-randomization-power.tex

```diff
@@ -391,37 +730,39 @@ \section{Power calculation and randomization inference}
 similarly, you can find some that have lower-than-average values.
 Your sample or randomization will inevitably fall in one of these categories,
 and we need to assess the likelihood and magnitude of this occurrence.\sidenote{
-  \url{https://davegiles.blogspot.com/2019/04/what-is-permutation-test.html}}
+	\url{https://davegiles.blogspot.com/2019/04/what-is-permutation-test.html}}
 Power calculation and randomization inference are the two key tools to doing so.
 
-\subsection{Power calculations}
 
-Power calculations report the likelihood that your experimental design
-will be able to detect the treatment effects you are interested in.
+% why to do power calculations
+\textbf{Power calculations} report the likelihood that your experimental design
+\index{power calculations}
+will be able to detect the treatment effects you are interested in.\sidenote{
+	\url{https://dimewiki.worldbank.org/Sample\_Size\_and\_Power\_Calculations}}
 This measure of \textbf{power} can be described in various different ways,
 each of which has different practical uses.\sidenote{
-  \url{https://www.stat.columbia.edu/~gelman/stuff_for_blog/chap20.pdf}}
+	\url{https://www.stat.columbia.edu/~gelman/stuff_for_blog/chap20.pdf}}
 The purpose of power calculations is to identify where the strengths and weaknesses
 of your design are located, so you know the relative tradeoffs you will face
 by changing your randomization scheme for the final design.
-They also allow realistic interpretations of evidence:
-low-power studies can be very interesting,
-but they have a correspondingly higher likelihood
-of reporting false positive results.
 
+
+% intuition of power concets: MDE and minimum sample size
 The classic definition of power
 is the likelihood that a design detects a significant treatment effect,
 given that there is a non-zero treatment effect in reality.
-This definition is useful retrospectively,
-but it can also be re-interpreted to help in experimental design.
 There are two common and useful practical applications
 of that definition that give actionable, quantitative results.
```

Comment:
what are the two? this is a nice topic sentence but then no follow-up!

Reply:
@bbdaniels , you wrote this. Are the applications _minimum-detectable-effect_ and _minimum-sample-size_?